### PR TITLE
add wo mode

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

File: logpyle/__init__.py [hunk 1]
```diff
@@ -389,7 +389,8 @@ def __init__(self, filename=None, mode="r", mpi_comm=None, capture_warnings=True
           If this database exists, the current state is loaded from it.
         :param mode: One of "w", "r" for write, read. "w" assumes that the
           database is initially empty. May also be "wu" to indicate that
-          a unique filename should be chosen automatically.
+          a unique filename should be chosen automatically. May also be "wo"
+          to indicate that the file should be overwritten.
```
Comment (inducer) [inline logpyle/__init__.py:393]: How is `"w"` different from `"wo"`?

**Review by inducer — COMMENTED** [review]
Thanks for this change. Generally, I'd like an opportunity to review these changes...

File: logpyle/__init__.py (hunk 1 above)
Comment (matthiasdiener) [reply logpyle/__init__.py:393]: `"w"` will not overwrite an existing file if it already exists (see line 490), `"wo"` will.

**Comment by matthiasdiener** [comment]
> Thanks for this change. Generally, I'd like an opportunity to review these changes...

Sorry, will add you as a reviewer from now on.

File: logpyle/__init__.py (hunk 1 above)
Comment (inducer) [reply logpyle/__init__.py:393]: OIC

**Pull request merged**